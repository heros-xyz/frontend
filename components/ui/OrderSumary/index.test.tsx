import { render } from "@testing-library/react";
import OrderSummary from ".";

it("renders homepage unchanged", () => {
  const { container } = render(
    <OrderSummary
      userName="Matt Prior"
      tier="Bronze tier"
      payment="$5.00"
      dateRenew="22 Jan 2023"
    />
  );
  expect(container).toMatchSnapshot();
});
