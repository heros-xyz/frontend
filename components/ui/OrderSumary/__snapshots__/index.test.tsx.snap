// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-1ra5f7y"
    payment="$5.00"
  >
    <h2
      class="chakra-heading css-qp0tw6"
    >
      Order Summary
    </h2>
    <div
      class="css-4u0ish"
    >
      <img
        alt="image"
        class="chakra-image css-n0pcsx"
        src=""
      />
      <div
        class="css-9knh52"
      >
        <span
          class="chakra-heading css-sde7o1"
        >
          Matt Prior
        </span>
        <p
          class="chakra-text css-1mqqk5k"
        >
          Bronze tier
        </p>
      </div>
      <div
        class="css-17xejub"
      />
      <p
        class="chakra-text css-1rxz9h8"
      />
    </div>
    <div
      class="css-1xp3c0"
    >
      <p
        class="chakra-text css-16579vr"
      >
        Monthly payment
      </p>
      <div
        class="css-17xejub"
      />
      <p
        class="chakra-text css-fzquwf"
      />
    </div>
    <div
      class="css-7qf1zq"
    >
      <p
        class="chakra-text css-0"
      >
        Today Charge
      </p>
      <div
        class="css-17xejub"
      />
      <p
        class="chakra-text css-1rxz9h8"
      />
    </div>
    <p
      class="chakra-text css-uuqzzv"
    >
      Renews automatically on 
      22 Jan 2023
    </p>
    <div
      class="css-16hv2lv"
    >
      <span
        class="chakra-text css-0"
      >
        By clicking subscribe now, you agree to our 
      </span>
      <a
        href="undefined/term-of-service"
        target="_blank"
      >
        <span
          class="chakra-text css-1081t4c"
        >
          Terms of User
        </span>
      </a>
      <span
        class="chakra-text css-0"
      >
         and 
      </span>
      <a
        href="undefined/privacy-policy"
        target="_blank"
      >
        <span
          class="chakra-text css-1081t4c"
        >
          Privacy Policy
        </span>
      </a>
      <span
        class="chakra-text css-0"
      >
        , and that this subscription automatically renews monthly. You'll be notified in advance if the monthly mount increases. Cancel anytime in your
         
      </span>
      <a
        href="/fan/active-subscriptions"
        target="_blank"
      >
        <span
          class="chakra-text css-1081t4c"
        >
          membership settings
        </span>
      </a>
      .
    </div>
  </div>
</div>
`;
